psi4/src/psi4/libfock/jk.h

```diff
@@ -1196,9 +1196,24 @@ class PSI_API DFJCOSK : public JK {
     int nthreads_;
     /// Options object
     Options& options_;
+
+    /// Perform Incremental Fock Build for J and K Matrices? (default false)
+    bool incfock_;
+      /// The number of times INCFOCK has been performed (includes resets)
+    int incfock_count_;
+    bool do_incfock_iter_;
+
     /// Previous iteration pseudo-density matrix
     std::vector<SharedMatrix> D_prev_;
+    
+    // D_ref_, the effective pseudo-density matrix is either:
+    //   (1) the regular density: D_eff == D_lr = C_lo x C*ro
+    //   (2) the difference density: D_eff == dD_lr = (C_lo x C_ro)_{iter} - (C_lo x C_ro)_{iter - 1}
+    std::vector<SharedMatrix> D_ref_;
 
+    // Is the JK currently on a guess iteration
```

Comment:
I dug into this a little bit. For DF_SCF_GUESS=True runs, it shows as true on the first CONV direct SCF iteration and false otherwise. For runs using a BASIS_GUESS, it shows up as true for the first iteration of each SCF cycle present (i.e., once for both the smaller and the larger basis set SCFs) and false otherwise. This is all from my testing. 

Further looking into it, this is a feature of the old DirectJK/DFJLinK IncFock scheme that does two things. First, that disables incremental Fock construction on the first SCF iteration of a given cycle. Second, it is set to true when the size of the previous D matrix vector is not the same length as the current D matrix vector. The second component is described in a comment in the original IncFock scheme in DirectJK as being "used to handle stability analysis case". I couldn't tell you how or why that particular component works, but I carried it over to make sure nothing broke.

Regardless, you're right; the comment could use a better description. I'm thinking something like `// Is the JK currently on the first SCF iteration of this SCF cycle?`